app/views/users/index.html.erb

```diff
@@ -1,7 +0,0 @@
-<%= link_to new_user_post_path(@user), class: 'card create flex' do %>
-  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-    <path fill="#696969" d="M11 19v-6H5v-2h6V5h2v6h6v2h-6v6h-2Z"></path>
-  </svg>
-  <span>Create New Post</span>
-<% end %>
-<%= users_list(@users) %>
```

Comment:
- [ ] **[OPTIONAL]** After logging in I get this error page. The whole app still working would be great, in my opinion. This is only optional since it is not a listed requirement of this very project but, as you can see, it is important that your app still works. Kindly bring back this file and other related ones. You deleted all posts and users views.

  ![image](https://github.com/c00p75/blog-rails/assets/81468675/54d8b194-0624-4e40-ac17-7b2cd6b2cbcb)